js/src/components/app-tooltip/index.js

```diff
@@ -3,12 +3,34 @@
  */
 import { Tooltip } from '@wordpress/components';
 import { Children } from '@wordpress/element';
+import { isWpVersion } from '@woocommerce/settings'; // eslint-disable-line import/no-unresolved
 
 /**
  * Internal dependencies
  */
 import './index.scss';
 
+// It's an inverse map of the following reference and contains only the values used in this repo.
+// Ref: https://github.com/WordPress/gutenberg/blob/wp/6.4/packages/components/src/popover/utils.ts#L17-L71
+const PLACEMENT_TO_POSITION = {
+	'top-start': 'top right',
+	top: 'top center',
+};
+
+// compatibility-code "WP < 6.4" -- `position` prop in Tooltip is deprecated since WP 6.4
+function mapPlacementToPosition( props ) {
+	if ( isWpVersion( '6.4', '<' ) ) {
```

Comment:
This will be always false as I read here

Reply:
The result will depend on the actual version of installed WordPress, not the one mocked in `js/src/tests/dependencies/woocommerce/settings.js`.